tests/queries/tests.py

```diff
@@ -280,6 +280,14 @@ def test_order_by_join_unref(self):
         qs = qs.order_by('id')
         self.assertNotIn('OUTER JOIN', str(qs.query))
 
+    def test_get_clears_ordering(self):
+        """
+        .get() should clear ordering for optimization purpose.
+        """
+        with CaptureQueriesContext(connection) as captured_queries:
+            Author.objects.order_by('name').get(pk=self.a1.pk)
+        self.assertNotIn('order by', captured_queries[0]['sql'].lower())
```

Comment:
could we inspect `query.order_by` instead? Maybe it's fine as-is, but that seems a bit less fragile.